### ISSUE TITLE
Feature request/question: Konami Code?

### ISSUE THREAD

**Issue by Noorquacker** [MultiMC/MultiMC5#2469]
I was digging and making a futile attempt at making MultiMC's launch process make sense in my head, when I stumbled upon KonamiCode and then looked and found this in MainWindow.cpp, line 828:
`// ENV.enableFeature("NewModsPage");`
I have no idea what it did and didn't bother to look further. Uncommenting it and seeing what happens didn't result in anything noticeable other than a Super Secret message, then after looking it up, I found out what the Konami Code is, and I'm thinking, can we get it back?

**TL;DR** bring back the Konami Code!

**Comment by peterix** [comment]
Well, I'll find a use for it at some point :) Soon:tm:
